modules/ROOT/pages/updates-upgrades-rollbacks.adoc

```diff
@@ -30,25 +30,36 @@ Alternatively, to check for available updates without downloading them, run:
 Upgrading between major versions (such as from Fedora 35 to Fedora 36) can be completed using the Software application.
 Alternatively, {variant-name} can be upgraded between major versions using the `ostree` command.
 
-First, verify the branch is available.
-You can print all available branches with this command:
+NOTE: Skipping major releases is not supported on Silverblue.
```

Comment:
Given the added context about lack of testing, could you add a small statement about that here?